core/src/main/resources/error/error-classes.json

```diff
@@ -564,6 +570,12 @@
     ],
     "sqlState" : "42000"
   },
+  "UNPIVOT_VALUE_SIZE_MISMATCH" : {
+    "message" : [
+      "All unpivot value columns must have the same size as there are value column names (<names>): [<sizes>]"
```

Comment:
We probably don't need to put `<sizes>`, as it's very clear from the SQL statement.